src/Routes.tsx

```diff
@@ -39,63 +38,61 @@ export function RoutedApp() {
       localStorage.setItem('landed', 'true');
       history.push(APP_ROUTES.about);
     }
-  }, [loading, userModel]);
+  }, [userModel]);
 
   return (
     <StrictMode>
       <OriginalLogo />
 
       <Router history={history}>
         <CommonRoutesFeatures userModel={userModel}>
-          {userModel && !loading && (
-            <CacheSwitch>
-              <CacheRoute
-                exact
-                path={[APP_ROUTES.map, APP_ROUTES.eventExplorer]}
-                className={s.mainViewWrap}
-              >
-                <Suspense fallback={null}>
-                  <MainView userModel={userModel} />
-                </Suspense>
-              </CacheRoute>
-
-              <Route exact path={APP_ROUTES.reports}>
-                <Suspense fallback={null}>
-                  <Reports />
-                </Suspense>
-              </Route>
+          <CacheSwitch>
+            <CacheRoute
+              exact
+              path={[APP_ROUTES.map, APP_ROUTES.eventExplorer]}
+              className={s.mainViewWrap}
+            >
+              <Suspense fallback={null}>
+                <MainView userModel={userModel} />
+              </Suspense>
+            </CacheRoute>
 
-              <Route exact path={APP_ROUTES.about}>
-                <Suspense fallback={null}>
-                  <AboutPage />
-                </Suspense>
-              </Route>
+            <Route exact path={APP_ROUTES.reports}>
+              <Suspense fallback={null}>
+                <Reports />
+              </Suspense>
+            </Route>
 
-              <Route path={APP_ROUTES.reportPage}>
-                <Suspense fallback={null}>
-                  <ReportPage />
-                </Suspense>
-              </Route>
+            <Route exact path={APP_ROUTES.about}>
+              <Suspense fallback={null}>
+                <AboutPage />
+              </Suspense>
+            </Route>
 
-              <Route path={APP_ROUTES.bivariateManager}>
-                <Protected
-                  pass={userModel.hasFeature(AppFeature.BIVARIATE_COLOR_MANAGER)}
-                >
-                  <Suspense fallback={null}>
-                    <BivariateManagerPage />
-                  </Suspense>
-                </Protected>
-              </Route>
+            <Route path={APP_ROUTES.reportPage}>
+              <Suspense fallback={null}>
+                <ReportPage />
+              </Suspense>
+            </Route>
 
-              <Route path={APP_ROUTES.profile}>
+            <Route path={APP_ROUTES.bivariateManager}>
+              <Protected
+                pass={!!userModel?.hasFeature(AppFeature.BIVARIATE_COLOR_MANAGER)}
```

Comment:
oh, let's make a pass: boolean; -> pass?: boolean; to get rid of `!!`
it still does a boolean convertion inside of <Protected/>